.travis.yml

```
@@ -14,10 +14,9 @@ matrix:
   allow_failures:
     - d: dmd-nightly
   include:
-    - d: ldc-1.11.0
+    - d: ldc
```

```suggestion
    - d: ldc-1.15.0
```

Let's lock this to avoid any sudden deployment errors (happened in the past).

---

Done